Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -794,17 +794,17 @@ - (void)createBannerWithTitle:(NSString *)title bannerColor:(UIColor *)bannerCol
     [bannerView addSubview:closeButton];
     const CGFloat kBannerCloseButtonPadding = 8.f;
     [closeButton autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:kBannerCloseButtonPadding];
-    [closeButton autoPinEdgeToSuperviewEdge:ALEdgeRight withInset:kBannerCloseButtonPadding];
```

Comment:
"Close" button is outside the banner in LTR

<img width="259" alt="screen shot 2017-07-20 at 12 25 25 pm" src="https://user-images.githubusercontent.com/217057/28428196-8fec3458-6d46-11e7-9f3a-3b5a868620b8.png">

Both LTR and RTL look fine if you take the original and only change `ALEdgeRight` -> `ALEdgeTrailing`:

e.g. 
```
[closeButton autoPinEdgeToSuperviewEdge:ALEdgeTrailing withInset:kBannerCloseButtonPadding];
```